packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -973,27 +894,104 @@ void main() {
     fail('Setting headerFile that does not exist should fail');
   });
 
-  testUsingContext('setting useDefferedLoading to null should fail', () {
-    _standardFlutterDirectoryL10nSetup(fs);
+  group('generateLocalizations', () {
```

Comment:
These two tests were rewritten based on the two removed from `build_system/targets/localizations_test.dart`, but without mocks.